frontend/src/app/patients/AddPatient.test.tsx

```diff
@@ -46,19 +48,19 @@ const fillOutForm = (
   }
 ) => {
   Object.entries(inputs).forEach(([label, value]) => {
-    userEvent.type(
+    fireEvent.change(
```

Comment:
ditto to comment above on why we're using fireEvent over userEvent

Reply:
same answer as before!